cob_description/urdf/cob4_gripper/gripper.urdf.xacro

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find cob_description)/urdf/cob4_gripper/gripper.gazebo.xacro" />
+  <xacro:include filename="$(find cob_description)/urdf/cob4_gripper/gripper.transmission.xacro" />
+
+  <xacro:macro name="gripper" params="name parent *origin finger_1_ref finger_2_ref">
+
+    <!-- joint between base_link and gripper_base_link -->
+    <joint name="${name}_base_joint" type="fixed" >
+      <insert_block name="origin" />
+      <parent link="${parent}" />
+      <child link="${name}_base_link" />
+    </joint>
+
+    <link name="${name}_base_link">
+      <xacro:default_inertial/>
+<!--
+       <xacro:cylinder_inertial radius="0.300" length="0.046" mass="5.0">
+        <origin xyz="0.03 0.08 0.01" rpy="0 0 0" />
+      </xacro:cylinder_inertial>
+-->
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <box size="0.001 0.001 0.001" />
+        </geometry>
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <box size="0.001 0.001 0.001" />
+        </geometry>
+      </collision>
+    </link>
+
+
+    <!-- joint between base_link and gripper_palm_link -->
+    <joint name="${name}_palm_joint" type="fixed">
+      <origin xyz="0 0 0" rpy="0 0 0" />
+      <parent link="${name}_base_link"/>
+      <child link="${name}_palm_link"/>
+    </joint>
+
+    <link name="${name}_palm_link">
+      <xacro:default_inertial/>
+      <visual>
+        <origin xyz="0 0 0.005" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://cob_description/meshes/cob4_gripper/palm.stl" />
+        </geometry>
+        <material name="IPA/LightGrey" />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0.0005" rpy="0 0 0" />
```

Comment:
where do these offsets come from?
is this offset intended or from copy-and-paste?